boards/arm/stm32f072_eval/doc/stm32f072_eval.rst

```diff
@@ -0,0 +1,195 @@
+- VDD from 2.0 V to 3.6 V
+- 128 KB Flash
+- 16 KB SRAM with HW parity
+- GPIO with external interrupt capability
+- one 12-bit ADC with 16 channels
+- one 12-bit D/A converters with 2 channels
+- RTC
+- Advanced-control Timer
+- General Purpose Timers (8)
+- Watchdog Timers (2)
+- USART (4)
+- I2C (2)
+- SPI (2)
+- CAN
+- USB 2.0 OTG FS with on-chip PHY
+- CRC calculation unit
+- DMA Controller
+- HDMI CEC Controller
+- 24 capacitive sensing channels for touchkey, linear and rotary touch sensors
+- Up to 87 fast I/Os, 68 I/Os with 5V tolerant capability and 19 with independent supply
+
+More information about STM32F072VB can be found here:
+       - `STM32F072VB on www.st.com`_
+       - `STM32F072 reference manual`_
+
+Supported Features
+==================
+
+The Zephyr stm32f072_eval board configuration supports the following hardware features:
+
++-----------+------------+-------------------------------------+
+| Interface | Controller | Driver/Component                    |
++===========+============+=====================================+
+| NVIC      | on-chip    | nested vector interrupt controller  |
++-----------+------------+-------------------------------------+
+| UART      | on-chip    | serial port-polling;                |
+|           |            | serial port-interrupt               |
++-----------+------------+-------------------------------------+
+| PINMUX    | on-chip    | pinmux                              |
++-----------+------------+-------------------------------------+
+| GPIO      | on-chip    | gpio                                |
++-----------+------------+-------------------------------------+
+| PWM       | on-chip    | pwm                                 |
++-----------+------------+-------------------------------------+
+| CLOCK     | on-chip    | reset and clock control             |
++-----------+------------+-------------------------------------+
+| IWDG      | on-chip    | independent watchdog                |
++-----------+------------+-------------------------------------+
+
+Other hardware features are not yet supported on Zephyr porting.
+
+The default configuration can be found in the defconfig file:
+
+	``boards/arm/stm32f072_eval/stm32f072_eval_defconfig``
+
+
+Pin Mapping
+===========
+
+STM32F072-EVAL Discovery kit has 6 GPIO controllers. These controllers are responsible for pin muxing,
+input/output, pull-up, etc.
+
+For mode details please refer to `STM32F072-EVAL board User Manual`_.
+
+Default Zephyr Peripheral Mapping:
+----------------------------------
+- UART_2_TX : PD5
+- UART_2_RX : PD6
+- TAMPER_PB : PC13
+- JOYSTICK_RIGHT_PB : PE3
+- JOYSTICK_LEFT_PB : PF2
+- JOYSTICK_UP_PB : PF9
+- JOYSTICK_DOWN_PB : PF10
+- JOYSTICK_SEL_PB : PA0
+- LD1 : PD8
+- LD2 : PD9
+- LD3 : PD10
+- LD4 : PD11
+
+System Clock
+============
+
+STM32F072-EVAL System Clock could be driven by internal or external oscillator,
+as well as main PLL clock. By default System clock is driven by PLL clock at 48MHz,
```

Comment:
change to "the main PLL clock", "the System clock", and "the PLL clock"

Reply:
Done